server/routes/article.js

```diff
@@ -0,0 +1,79 @@
+const express = require('express');
+const router = express.Router();
+const articleLibs = require('../libs/articlelibs');
+const {authenticate} = require('../middleware/authenticate');
+
+router.post('/article/newArticle', (req, res) => {
+    let data = {
+        body : req.body,
```

Comment:
fields not body extract value from the body or you can do like data = req.body

Reply:
Done